tst/EnergyPlus/unit/SolarShading.unit.cc

```diff
@@ -3879,9 +3879,21 @@ TEST_F(EnergyPlusFixture, SolarShadingTest_Warn_Pixel_Count_and_TM_Schedule)
 
     SolarShading::GetShadowingInput(*state);
 
-    EXPECT_EQ(state->dataErrTracking->TotalWarningErrors, 0);
-    // Now expect one severe error from GetShadowInput()
-    EXPECT_EQ(state->dataErrTracking->TotalSevereErrors, 1);
-    // There should be a severe warning reported about the PixelCounting and the scheduled shading surface tm values > 0.0 combination.
-    EXPECT_EQ(state->dataErrTracking->LastSevereError, "The Shading Calculation Method of choice is \"PixelCounting\"; ");
+#ifdef EP_NO_OPENGL
+    EXPECT_EQ(state->dataErrTracking->TotalWarningErrors, 1);
+    EXPECT_EQ(state->dataErrTracking->TotalSevereErrors, 0);
+    EXPECT_EQ(state->dataErrTracking->LastSevereError, "");
+#else
```

Comment:
So now the directive is set, so we are going to attempt to use pixel counting.  However, if we don't get a valid context, then once again we issue a warning and set it back to polygon clipping.  Finally, if we actually get a valid context, the unit test should issue the actual severe.